WindowsServerDocs/remote/remote-desktop-services/clients/change-listening-port.md

```diff
@@ -22,3 +22,30 @@ When you connect to a computer (either a Windows client or Windows Server) throu
 6. Close the registry editor, and restart your computer.
 
 The next time you connect to this computer by using the Remote Desktop connection, you must type the new port. If you're using a firewall, make sure to configure your firewall to permit connections to the new port number.
+
+
+You can check the current port by running the following PowerShell command:
+
+```powershell
+Get-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp' -name "PortNumber"
+
```

Comment:
Also missing a code block fence end, 3 backticks in a new line needed.
~~~suggestion
```

~~~~